### ISSUE TITLE
I am  doing automation testing on the app which launches url inside (webview2 ) with login info ,  how to connect with that 

### ISSUE THREAD

**Issue by pooja1233** [MicrosoftEdge/WebView2Feedback#2238]
I tried open browser but it opens url inside but with different instance , if I click manually on that it load webpage , how to do that through automation

**Comment by champnic** [comment]
Hey @pooja1233 - Can you read the instructions on this page and see if it helps?
https://docs.microsoft.com/en-us/microsoft-edge/webview2/how-to/webdriver#approach-2-attaching-microsoft-edge-driver-to-a-running-webview2-app

If not, please let me know.

Thanks!

**Comment by champnic** [comment]
Your code looks a bit different then what's described for the options here:
https://docs.microsoft.com/en-us/microsoft-edge/webview2/how-to/webdriver#attaching-microsoft-edge-driver-to-the-launched-webview2-app

What version of Selenium are you using?

**Comment by pooja1233** [comment]
I have 3.141.0
I am using robot framework

**Comment by champnic** [comment]
I'm unfamiliar with the robot framework, but I believe you need to be using Selenium version 4.0+.

**Comment by pooja1233** [comment]
I did update selenium to 4.0 but still its not working , do you have any example in python hoe to connect to existing session , or how to open an webview app with url , I was trying to use  Zoomba.GUILibrary open browser but it again use another instance

**Comment by pooja1233** [comment]
I can grab process id pf webview app and inside there is webview2.exe , i can get port number too , I tried to connect through port number too but no luck

**Comment by champnic** [comment]
I don't think we have specific examples of this scenario, besides what's in the doc I posted. @bwalderman Any ideas?

**Comment by bwalderman** [comment]
Hi @pooja1233 , please remove `options.use_chromium` which is no longer necessary in Selenium 4. Please also remove `options.use_webbiew` and `options.binary_location`. These options should only be used if you want WebDriver to launch your app. Lastly, in Python, the name of the option to use for attaching is `debugger_address` instead of `DebuggerAddress`: https://www.selenium.dev/selenium/docs/api/py/webdriver_chromium/selenium.webdriver.chromium.options.html#module-selenium.webdriver.chromium.options

Here's an updated sample which should work:

```py
def create_edge():
options = EdgeOptions()
options.debugger_address = 'localhost:9222'
driver = Edge(options = options)
```
